solutions/baseline/main.tf

```diff
@@ -0,0 +1,81 @@
+#######################################################################################################################
+# Resource Group
+#######################################################################################################################
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  resource_group_name          = var.use_existing_resource_group == false ? ((var.prefix != null && var.prefix != "") ? "${var.prefix}-${var.resource_group_name}" : var.resource_group_name) : null
+  existing_resource_group_name = var.use_existing_resource_group == true ? var.resource_group_name : null
+}
+
+########################################################################################################################
+# OCP VPC cluster (single zone)
+########################################################################################################################
+
```

Comment:
There needs to be code to support optionally creating a KMS key in an existing KMS instance incase user wants to enable KMS encryption